src/crab/array_domain.cpp

```
@@ -206,26 +207,34 @@ std::vector<Cell> offset_map_t::get_overlap_cells(const offset_t o, const unsign
+                    dst.insert_cell(c);
+                }
+            }
+        }
+    }
+};
 
-void clear_stack_cell_registry(StackCellRegistry& registry) { registry.clear(); }
+std::shared_ptr<StackCellRegistry> make_stack_cell_registry() { return std::make_shared<StackCellRegistry>(); }
 
-void ArrayDomain::initialize_numbers(StackCellRegistry& cells, const int lb, const int width) {
+void ArrayDomain::initialize_numbers(const int lb, const int width) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'initialize_numbers' of similar type ('const int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)